shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -72,7 +74,7 @@
  * <p>Deprecation: {@link io.flutter.embedding.android.FlutterView} is the new API that now replaces
  * this class. See https://flutter.dev/go/android-project-migration for more migration details.
  */
-public class FlutterView extends SurfaceView implements BinaryMessenger, TextureRegistry {
+public class FlutterView extends SurfaceView implements BinaryMessenger, TextureRegistry, MouseCursorPlugin.MouseCursorViewDelegate {
```

Comment:
Is this missing an implementation of `setPointerIcon`?

Reply:
`setPointerIcon` is actually a method of Android `View`, which (I think) is inherited by `FlutterView`. This interface is used to limit the ability of `MouseCursorController` to control `FlutterView`, a response to https://github.com/flutter/engine/pull/13152#discussion_r335716998 (as far as I understand; please point out if there's a better way or it's unnecessary).